### PR TITLE
IW-2633 | decouple wgArchiveWikiForums from wgEnableForumExt

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/IW-2633

@Wikia/iwing

### CODE REVIEW

File: includes/wikia/Extensions.php [hunk 1]
```diff
@@ -796,7 +796,6 @@
 
 // Enable new style forums (/wiki/Special:Forum)
 if ( !empty( $wgEnableForumExt ) ) {
-	$wgArchiveWikiForums = true;
```
Comment (mszabo-wikia) [inline includes/wikia/Extensions.php:799]: If `$wgArchiveWikiForums` is not set in WF, we should still disable wiki style forums.

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: includes/wikia/Extensions.php (hunk 1 above)
Comment (rybmat) [reply includes/wikia/Extensions.php:799]: good point!

Comment (rybmat) [reply includes/wikia/Extensions.php:799]: so If I got it correctly, `ArchiveWikiForums` should by default be set to true, right?

Comment (mszabo-wikia) [reply includes/wikia/Extensions.php:799]: Nope, that would disable wiki forums for wikis without the threaded forums extension :P I suggest setting the variable to default `null` instead. Then, this block, we can check the value and only set it to true if the value is `null`, which indicates that it was not overridden in WF.

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**